tests/python/test-5-operator.py

```diff
@@ -1498,3 +1498,100 @@ def test_524(ceed_resource):
         assert abs(total - 1.0) < 1E-10
 
 # -------------------------------------------------------------------------------
+# Test assembly of mass matrix operator diagonal
```

Comment:
Not concerning this comment in particular, but while at it, would you please double check all others comments from earlier tests in this series? Some of them appear to be the same string for tests that do slightly different things.

Reply:
I updated some to better descriptions.

Reply:
Thanks! Some of them still have the same descriptions, but I can't spot at a first glance now what are the differences in those tests

Reply:
They are the same test, but with 0 vs 1 as the input vector